src/StoreKeeper/WooCommerce/B2C/Updator.php

```diff
@@ -97,5 +97,12 @@ private function upgradeDatabase(string $databaseVersion): void
                 $migrate->run();
             }
         }
+
+        if (version_compare($databaseVersion, '7.4.6', '<')) {
+            $currentPaymentGatewayOption = StoreKeeperOptions::get(StoreKeeperOptions::PAYMENT_GATEWAY_ACTIVATED);
+            if ('no' === $currentPaymentGatewayOption || is_null($currentPaymentGatewayOption)) {
```

Comment:
```suggestion
            if ( is_null($currentPaymentGatewayOption)) {
```
if it was disabled than you should not change it

Reply:
Implemented.